core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -93,6 +93,7 @@ public DockerComposeContainer(String identifier, List<File> composeFiles) {
 
         // Use a unique identifier so that containers created for this compose environment can be identified
         this.identifier = identifier;
+        project = identifier + Base58.randomString(6).toLowerCase();
```

Comment:
without this change, `compose.start(); compose.stop(); compose.start()` would use the same `randomIndentifier` and might lead to some unpredictable side effects